src/stravalib/protocol.py

```diff
@@ -193,17 +299,23 @@ def exchange_code_for_token(
                 "grant_type": "authorization_code",
             },
             method="POST",
+            refresh_expired_token=False,
```

Comment:
avoids an infinite loop. i haven't triggered one of these in a long time until I worked on this pr 😆